api/desecapi/exceptions.py

```diff
@@ -1,17 +1,28 @@
 import json
 from json import JSONDecodeError
 
-from rest_framework.exceptions import APIException
+from rest_framework import status
+from rest_framework.exceptions import APIException, ValidationError
 
 
-class PDNSException(APIException):
+class RequestEntityTooLarge(APIException):
+    status_code = status.HTTP_413_REQUEST_ENTITY_TOO_LARGE
+    default_detail = 'Payload too large.'
+    default_code = 'too_large'
+
+
+class PDNSValidationError(ValidationError):
+    status_code = 422
```

Comment:
should we change this code to 400? Our own validation will also return 400, so why differ here by where the error came from?